src/box/iproto.cc

```diff
@@ -3395,6 +3395,11 @@ struct iproto_cfg_msg: public cbus_call_msg
 			/** Whether the request handler is set or reset. */
 			bool is_set;
 		} override;
+		/**
+		 * Connection that executing iproto_drop_connections.
+		 * NULL if the function is called not from connection.
+		 */
+		struct iproto_connection *drop_connection;
```

Comment:
Bad name. Better be:

```C
struct {
        /** ... */
        struct iproto_connection *owner;
} drop_connections;
```

Reply:
Fixed.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/iproto.cc b/src/box/iproto.cc
index cc9433a0b..6b26f1d6e 100644
--- a/src/box/iproto.cc
+++ b/src/box/iproto.cc
@@ -3395,11 +3395,13 @@ struct iproto_cfg_msg: public cbus_call_msg
 			/** Whether the request handler is set or reset. */
 			bool is_set;
 		} override;
-		/**
-		 * Connection that executing iproto_drop_connections.
-		 * NULL if the function is called not from connection.
-		 */
-		struct iproto_connection *drop_connection;
+		struct {
+			/**
+			 * Connection that executing iproto_drop_connections.
+			 * NULL if the function is called not from connection.
+			 */
+			struct iproto_connection *owner;
+		} drop_connections;
 	};
 	struct iproto_thread *iproto_thread;
 };
@@ -3504,7 +3506,7 @@ iproto_do_cfg_f(struct cbus_call_msg *m)
 			 * Do not wait deletion of connection that called
 			 * iproto_drop_connections to avoid deadlock.
 			 */
-			if (con != cfg_msg->drop_connection) {
+			if (con != cfg_msg->drop_connections.owner) {
 				con->is_drop_pending = true;
 				iproto_thread->drop_pending_connection_count++;
 			}
@@ -3586,17 +3588,16 @@ iproto_drop_connections(void)
 {
 	static struct latch latch = LATCH_INITIALIZER(latch);
 	latch_lock(&latch);
-	struct iproto_connection *drop_connection = NULL;
+	struct iproto_connection *owner = NULL;
 	struct session *session = fiber_get_session(fiber());
 	if (session != NULL && session->type == SESSION_TYPE_BINARY)
-		drop_connection =
-			(struct iproto_connection *)session->meta.connection;
+		owner = (struct iproto_connection *)session->meta.connection;
 	drop_pending_thread_count = iproto_threads_count;
 	for (int i = 0; i < iproto_threads_count; i++) {
 		struct iproto_cfg_msg *cfg_msg =
 			(struct iproto_cfg_msg *)xmalloc(sizeof(*cfg_msg));
 		iproto_cfg_msg_create(cfg_msg, IPROTO_CFG_DROP_CONNECTIONS);
-		cfg_msg->drop_connection = drop_connection;
+		cfg_msg->drop_connections.owner = owner;
 		iproto_do_cfg_async(&iproto_threads[i], cfg_msg);
 	}
 

```

</details>